Childrens-Social-Care-CPD/Views/Shared/_Header.cshtml

```diff
@@ -69,6 +69,10 @@
                     {
                         RenderMenuItem("agencyResources", "Agency resources", "agency-resources", category == "Agency resources");
                     }
+                    if (featuresConfig.IsEnabled(Features.PathwaysTraining))
+                    {
+                        RenderMenuItem("pathwaysTraining", "Pathways training", "pathways-social-work-leadership-modules-available", category == "Pathways training");
```

Comment:
Query: Is this the right URL for the top-level navigation?  Should it be the 'modules available' page or the 'all pathways overview' page.

Reply:
Amended